packages/material-ui-system/src/css.js

```diff
@@ -13,15 +13,32 @@ function omit(input, fields) {
   return output;
 }
 
+function handleCss(styleFunction, theme) {
+  const apply = styles => {
+    const cssStyles = omit(styles, styleFunction.filterProps);
+    const output = {
+      ...styleFunction({ theme, ...styles }),
+      ...cssStyles,
+    };
+
+    Object.keys(cssStyles).forEach(key => {
+      if (typeof cssStyles[key] === 'object') {
```

Comment:
This is `true` for `null`. If we actually want to handle `null` and objects, add a comment explaining this. Usually typeof comparisons with object are only concerned with plain objects.

Reply:
👍 Fixed it!